index.js

```diff
@@ -285,6 +295,24 @@ SendStream.prototype.isConditionalGET = function(){
 };
 
 /**
+ * Return the array of file precompressed file extensions to serve in preference order.
+ *
+ * @return {Array}
+ * @api private
+ */
+
+SendStream.prototype.getAcceptEncodingExtensions = function() {
+  var accepted = []
+  var header = this.req.headers['accept-encoding']
+  if (header) {
+    this._precompressionFormats.forEach(function (format) {
+      if (header.indexOf(format.encoding) >= 0) accepted.push(format.extension)
```

Comment:
Looks like this is broken for web browsers that send "no-gzip, deflate", which was common for a time (Apache, and I assume others, understand no-gzip).

This also does not seem to take into account the quality parameters at all, which is very common from CDNs, as they usually want a specific encoding to store.